Kernel/Prekernel/multiboot.S

```diff
@@ -14,7 +14,7 @@
 .long multiboot_checksum
 
 
-/* for MULTIBOOT_MEMORY_INFO */
+/* for MULTIBOOT_AOUT_KLUDGE, just a dummy */
```

Comment:
"Fix wrong and misleading kernel"? This is not understandable.
What you mean is that you change the comment to represent the given parameters (which are for a.out kludge).

Reply:
Sorry, got distracted and mistyped. Fixed